script-wrapper.sh

```diff
@@ -40,7 +37,7 @@ EOF
 # clean up wrapper afterwards, or else it leaves files owned by root in /workspace
```

Comment:
Now that we might be running this in `exec` mode with the user's working dir mounted in, do we still want to write the temporary wrapper script out next to the real script? Or should we stick it in `/tmp` instead? I know we clean up the wrapper but it still feels a bit icky to be using the working dir in this way.

I can't remember if there was a reason other than neatness that we originally ended putting in next to the real script. Maybe it affects what Stata thinks the CWD is?

If we do end up keeping it in the working dir, should we use `mktemp` rather than a fixed name so there's no possibility of blatting an existing file, or of simultaneous Stata runs interacting with each other somehow?

Reply:
I don't recall either, and had wondered the same. `stata -b` defaults to writing `$SCRIPT.log` in the same directory as `$SCRIPT.do`, but we explicitly emulate that anyway.

I changed and the tests still pass, so have added

Reply:
Great, looks good to me.